torchvision/prototype/features/_image.py

```diff
@@ -13,6 +13,7 @@
 
 class ColorSpace(StrEnum):
     # this is just for test purposes
+    # How about the transparency spaces supported by ImageReadMode?
```

Comment:
This enum are not complete yet and should be expanded as we mature the prototype.